src/ig/IGExporter.ts

```diff
@@ -178,6 +178,21 @@ export class IGExporter {
         value: `${this.config.canonical}/history.html`
       });
     }
+    // Default 'autoload-resources' to false if it is not already defined
+    // and only if custom resources are defined. This is done to counter
+    // the IG Publisher behaving as if 'autoload-resources' is set to true
+    // and allowing the IG Publisher to attempt processing of custom resources.
+    if (
+      this.pkg.resources?.length > 0 &&
+      !this.ig.definition.parameter.some(param => {
+        return param.code === 'autoload-resources';
+      })
+    ) {
```

Comment:
Can you put in a logging statement (`info`) indicating that `autoload-resources` is being set to false since this IG contains custom resources?